src/utilities/qsocket.c

```diff
@@ -112,8 +112,9 @@ bool qsocket_close(int sockfd, int timeoutms) {
         char buf[1024];
         while (true) {
             ssize_t read = qio_read(sockfd, buf, sizeof(buf), timeoutms);
+			DEBUG("Throw %zu bytes from dummy input stream.", read);
```

Comment:
seems the indentation is not lined up.

Reply:
qlibc uses spaces, please ignore using tabs

Reply:
Oops, didn't realize that, thanks for the heads up.